config/application.yml.default

```diff
@@ -587,6 +589,8 @@ test:
   verify_personal_key_attempt_window_in_minutes: 3
   verify_personal_key_max_attempts: 2
   usps_ipp_root_url: 'http://localhost:1000'
+  usps_ipp_sponsor_id: '111111111111111'
+  usps_eipp_sponsor_id: '222222222222222'
```

Comment:
I added these configs in the test section so that spec/factories/in_person_enrollments.rb will always have a value when they are not set in individual specs